wizards/Policy/PolicyWizard.tsx

```diff
@@ -338,6 +339,31 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
                         validation={isValidKubernetesResourceName}
                     />
 
+                    <WizRadioGroup
+                        path="objectDefinition.spec.pruneObjectBehavior"
+                        label="Prune Object Behavior"
+                        labelHelp="Optional. Specifies how related objects on the managed cluster should be pruned when the policy is removed. The parameter values are None, DeleteIfCreated, and DeleteAll."
+                    >
+                        <Radio
+                            id="deleteIfCreated"
+                            label="Delete If Created"
+                            value="DeleteIfCreated"
+                            description="Attempts to remove objects known to be created by the policy when the policy is deleted."
+                        />
+                        <Radio
+                            id="deleteAll"
+                            label="Delete All"
+                            value="DeleteAll"
+                            description="Attempts to remove all objects related to the policy when the policy is deleted."
+                        />
+                        <Radio
+                            id="none"
+                            label="None"
+                            value="None"
+                            description="Does not remove any resources when the policy is removed (default)."
```

Comment:
Does not delete any resources when the policy is deleted. This value is used by default.